core/lib/dal/src/models/proto/mod.proto

```diff
@@ -23,3 +16,75 @@ message Payload {
   repeated Transaction transactions = 8;
   optional bool last_in_batch = 10; // required
 }
+
+message Transaction {
+  oneof common_data {
+    L1TxCommonData L1 = 1;
+    L2TxCommonData L2 = 2;
+    ProtocolUpgradeTxCommonData ProtocolUpgrade = 3;
+  }
+  optional Execute execute = 4; // required
+  optional uint64 received_timestamp_ms = 5; // required
```

Comment:
Actually, there's an open question whether it makes sense to transmit / use this field on EN because doing so [seemingly makes](https://linear.app/matterlabs/issue/PLA-782/fix-pending-transactions-subcription-on-en) pending transaction subscriptions buggy. But yeah, right now this field is in use, so it's more of a JFYI comment.